web/src/assets/locales/en.json

```diff
@@ -1,6 +1,6 @@
 {
     "cancel": "Cancel",
-    "login": "Login",
+    "login": "Sign in",
```

Comment:
This is an unrelated change and I don't think I like it

Reply:
Done